pom.xml

```diff
@@ -16,10 +16,10 @@
 		<module>basyx.common</module>
 		<module>basyx.submodelservice</module>
 		<module>basyx.submodelrepository</module>
-		<module>basyx.submodelregistry</module>
+		<!--<module>basyx.submodelregistry</module>-->
```

Comment:
Just pointing to commented out modules :)

Reply:
Thanks for pointing this out, I'll correct this in the latest commit.